modular_skyrat/modules/customization/modules/language/xerxian.dm

```diff
@@ -0,0 +1,13 @@
+/datum/language/xerxian
+	name = "Xerxian"
+	desc = "An evolved form of the language of Arabic from Earth. Despite its ancient origins, it is still spoken by many people in cultures that evolved from the Middle-East."
```

Comment:
only other thing i'll note is that this could use a more open-ended description, but it'd be a new lore thing to explore so I can't reference anything else rn to use, nor even say you NEED to do it. Its something that'll need some discussion, though I see lots of potential for it, especially with a name that's not strictly tying it as starting on-earth.

Biggest recommendation is, again, open-endedness. Sure, it started on ancient earth, but to say its spoken by cultures evolved from the middle-east is almost restricting. See: the changes I requested for Selenian, namely the end, pointing out that while it started on Luna it had the capability to spread far and fast.

So, how might Xerxian have spread far and fast, if at all? What pushed it to space in the first place, and why did it stick? Perhaps it was part of a cultural/religious/economic/any kind of movement after the majority of earth was space-worthy, perhaps it was a partnership with companies, labor in exchange for 'cultural diversity'. _Perhaps they just started shooting rockets full of information into space and some poor sods found it and treated it as a holy tongue_ (Looking at Taroon here, those cursed people)
And then, how did it spread? Was it just the culture moving about the system, such as Sol and Selenian through the Milky way? Was it conquest and indoctrination? Or was it the same as the last point from before, it spread because it just got launched everywhere at random?